src/lib/netlist/devices/nld_4029.cpp

```diff
@@ -0,0 +1,283 @@
+// license:GPL-2.0+
+// copyright-holders:Couriersud, Jonathan Gevaryahu
+/*
+ * nld_4029.cpp
+ *
+ *  CD4029BM/CD4029BC Presettable Binary/Decade Up/Down Counter
```

Comment:
It would be cool to merge these comments with the "netlist" documentation in the macro folder.

Reply:
Merged in latest.